test cases/fortran/10 find library/meson.build

```diff
@@ -3,7 +3,7 @@ project('find fortran library', 'fortran')
 fortranc = meson.get_compiler('fortran')
 
 sources = ['main.f90', 'gzip.f90']
-zlib = fortranc.find_library('z')
```

Comment:
This used find_library on purpose to test that it is working (which is why the test directory is called `10 find library`).

Reply:
reverted